detect.go

```diff
@@ -210,6 +231,28 @@ func Detect(
 			})
 
 			requirements = append(requirements, packit.BuildPlanRequirement{
+				Name: "dotnet-core-aspnet-runtime",
+				Metadata: BuildPlanMetadata{
+					Launch: true,
+				},
+			})
+
+			requirements = append(requirements, packit.BuildPlanRequirement{
```

Comment:
This buildpack doesn't actually have to require the .NET SDK anymore, since the dotnet-publish buildpack [handles this](https://github.com/paketo-buildpacks/dotnet-publish/blob/96dd50cfe2dde53fc3b00cdc9e13ae558e7e373b/detect.go#L61) and the `dotnet` CLI installed in the dotnet-core-aspnet-runtime layer will be used at app launch time.

Reply:
I think that all instance of requiring the have been removed.